build/InstrumentationLibraries.proj

```diff
@@ -1,9 +1,19 @@
 <Project>
   <ItemGroup>
     <SolutionProjects Include="..\**\OpenTelemetry.Instrumentation*.csproj" />
+
+    <TestProjects Include="..\test\OpenTelemetry.Instrumentation*.Tests\*.csproj" />
   </ItemGroup>
 
   <Target Name="Build">
-    <MSBuild Projects="@(SolutionProjects)" Targets="Restore;Build" ContinueOnError="ErrorAndStop" />
+    <MSBuild Projects="@(SolutionProjects)" Targets="Build" ContinueOnError="ErrorAndStop" />
+  </Target>
+
+  <Target Name="Restore">
+    <MSBuild Projects="@(SolutionProjects)" Targets="Restore" ContinueOnError="ErrorAndStop" />
+  </Target>
+
+  <Target Name="VSTest">
```

Comment:
is this for local VS development?

Reply:
`dotnet test` CLI invokes this target for some reason.

Reply:
I ended up reverting this because dotnet test via the .proj would fail some of the projects without targets like `net462`. Had to go back to manual style: `dotnet test **/bin/Release/${{ matrix.version }}/OpenTelemetry.Instrumentation*.Tests.dll`